models/mongo/conversations.go

```diff
@@ -0,0 +1,63 @@
+// Package mongo is an implementation of models.AccountsRepository
+// that persists data on a MongoDB database.
+package mongo
+
+import (
+	"accounts-service/models"
+	"context"
+
+	// "errors"
+
+	// "github.com/google/uuid"
+	// "go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	// "go.mongodb.org/mongo-driver/mongo/options"
+	"go.uber.org/zap"
```

Comment:
You should configure your IDE so that imports are automatically generated. You can do this in VSCode by installing the Go extension and the go toolchain. This makes it so imports are sorted in the right order and the useless ones are removed.